### DISCUSSION TITLE
Is this a scam?

### DISCUSSION THREAD

**Discussion by bhagaban** [lllyasviel/Fooocus#3231] [Q&A] [answered]
When you search for Fooocus on Google, this website pops up.
![Screenshot 2024-07-03 at 6 43 46 PM](https://github.com/lllyasviel/Fooocus/assets/826722/29c1e5cc-8290-4265-bdb1-531b7c371d65)
Is this affiliated with this OS project? It offers infinite generations and  looks too good to be true. 
It has links to other scam links as well. Very well done website, has high rank on google - surely many would have lost money via this link.

**Comment by mashb1t** [comment]
It's not affiliated and it is a scam!
https://github.com/lllyasviel/Fooocus/issues/3200#issuecomment-2195725719

**Comment by mashb1t** [comment, marked as answer]
Updated the list of all (known) fake / scam websites and pinned the discussion => https://github.com/lllyasviel/Fooocus/discussions/2146
